src/templates/app/index.html

```diff
@@ -207,7 +207,7 @@
         <div class="app-more-metadata">
           <dl>
             <dt>{{ _('Developer') }}</dt>
-            <dd>{{ this.author }}</dd>
+            <dd><a href="{{ url('search')|urlparams(src='this.author') }}">{{ this.author }}</a></dd>
```

Comment:
You're almost there. However with this you are setting the `src` parameter, when you want to set the `author` parameter. So you need to adjust the `urlparams` call.